cli-access-protected-api/README.md

```diff
@@ -0,0 +1,112 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- java
+page_type: sample
+name: "Java console application written that accesses Microsoft Graph as itself"
+description: "This Java console application contacts Microsoft Graph as itself. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-webapp-java
+---
+-->
+
+# Java | console app | protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This sample demonstrates a Java console application that accesses Microsoft Graph as itself by using the Microsoft Authentication Library (MSAL) for Java.
+
+![A screenshot showing the output of executing the command above, which is JSON from a Graph API call.](./output.png)
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Java 11+
+- Maven
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting | Value for this sample app                                                    | Notes                                                                                              |
+|:------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                       | `java-cli`                                                                   | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**              | _None_                                                                       | Required value for this sample. |
```

Comment:
Went with the option to explicitly state it as you suggested.  The lack of detail there might make a customer wonder if they should select something (since they don't have perfect knowledge of our intent).  Thanks for the suggestion!